package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "webpack-plugin-hash-output",
-  "version": "1.3.0",
+  "version": "1.3.1",
```

Comment:
Can you drop this change please? It will mess with the release scripts I'll run once this change is merged.